core/src/main/kotlin/com/strumenta/kolasu/parsing/ParseTree.kt

```diff
@@ -70,6 +70,7 @@ fun ParserRuleContext.processDescendantsAndErrors(
     }
 }
 
+@Deprecated("Use KolasuParser")
```

Comment:
Good point

Reply:
I wasn't sure to delete it but the IDE flags it as duplicate code